DOC-756: Document series column feature for the advtable plugin (#1870)
tinymce/tinymce-docs: _includes/commands/advtable-cmds.md

| Command                  | Description                                                                                     |
| ------------------------ | ----------------------------------------------------------------------------------------------- |
| mceAdvancedTableSort     | Opens the Advanced Table Sort Dialog for the current selection or cursor location.              |
| mceSortTableAdvanced     | Performs an Advanced Table Sort. For details, see [mceSortTableAdvanced](#mcesorttableadvanced).                                         |
| mceSortTableByColumnAsc  | Sorts the current table ascending by column based on the current cursor position or selection.  |
| mceSortTableByColumnDesc | Sorts the current table descending by column based on the current cursor position or selection. |
| mceSortTableByRowAsc     | Sorts the current table ascending by row based on the current cursor position or selection.     |
| mceSortTableByRowDesc    | Sorts the current table descending by row based on the current cursor position or selection.    |

> | mceTableToggleSeries     | Toggles a series column on the selected table. For details, see [mceTableToggleSeries](#mcetabletoggleseries). {{ site.requires_5_7v }} |

Author: The code needs at least 5.7 and will work with 5.7 and above so I think this is fine.

| mceSortTableAdvanced     | Performs an Advanced Table Sort. See below for details.                                         |
| mceSortTableByColumnAsc  | Sorts the current table ascending by column based on the current cursor position or selection.  |
| mceSortTableByColumnDesc | Sorts the current table descending by column based on the current cursor position or selection. |
| mceSortTableByRowAsc     | Sorts the current table ascending by row based on the current cursor position or selection.     |
| mceSortTableByRowDesc    | Sorts the current table descending by row based on the current cursor position or selection.    |

> | mceTableToggleSeries     | Toggles a series column on the selected table. See below for more details. |

Reviewer: This will probably need the `{{ site.requires_5_7v }}` part here as well, to show it's only available from 5.7 onwards.

Reviewer 2: > See below for more details.

Change to form of "For details, see []()." 

It wont always be below....

- `sort` - `'row'`, `'column'`, `'selection'`, or `'table'`
- `order` - `'ascending'` or `'descending'`

`mceTableToggleSeries` accepts an object with the following key-value pairs:

> - `name` - a string indicating the series to toggle. Series and their associated names are configured using the [advtable_value_series]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series) option. If a different series name is provided than what is present on the table, the series column will be updated to use the new series.

Reviewer 3: ```suggestion
- `name` - a string indicating the series to toggle. Series and their associated names are configured using the [advtable_value_series]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series) option. If the provided series name is different to the name currently present on the table, the series column will be updated to use the new series.
```
Maybe something like this?

> - `location` - `'left'`

Reviewer 3: I feel like we should put a sentence here explaining that `'left'` isn't an example of one of the values you can put here, it's the only value you can put here. Then maybe say some words about the potential for forwards compatibility, if we even have anything to say there?

tinymce.activeEditor.execCommand('mceSortTableByColumnDesc')
tinymce.activeEditor.execCommand('mceSortTableByRowAsc')
tinymce.activeEditor.execCommand('mceSortTableByRowDesc')
tinymce.activeEditor.execCommand('mceTableToggleSeries', false, { name: 'numbers', location: 'left' })
```

### `mceSortTableAdvanced`

> `mceSortTableAdvanced` accepts an object with the following value pairs:

Reviewer: ```suggestion
`mceSortTableAdvanced` accepts an object with the following key-value pairs:
```
Maybe?

- `sortby` - `'row'` or `'column'`

> - `roworcol` - a zero-indexed integer in a string representing the row from the top of the table or column from the left of the table.

Reviewer 4: could this be called startindex or indexstart or just index?
perhaps mention this property is dependent on the 'sortby' property, (thats if we change the property to index)

Author: This is out of scope as this is related to `mceSortTableAdvanced` command. If I am looking at it correctly, it would have to be left as 'zero-indexed' as this indicates the first row or column is '0'.

tinymce.activeEditor.execCommand('mceSortTableByColumnDesc')
tinymce.activeEditor.execCommand('mceSortTableByRowAsc')
tinymce.activeEditor.execCommand('mceSortTableByRowDesc')
tinymce.activeEditor.execCommand('mceTableToggleSeries', false, { name: 'numbers', location: 'left' })
```

### `mceSortTableAdvanced`

> 

Author: This is an existing command so it doesn't need a 'requires' note. Unless you meant the other `mceTableToggleSeries` heading?

> 

Reviewer 2: do we know what version of TinyMCE is required for this? (5.1/5.2)?
